packages/neuron-ui/src/components/WalletWizard/index.tsx

```diff
@@ -24,7 +28,7 @@ export default (props: any) => {
   const wallet = useContext(WalletContext)
   useEffect(() => {
     const content = document.querySelector('.main-content')
-    if (content) {
+    if (content && !wallet) {
```

Comment:
is `content` necessary here? the wizard is mounted in it.

Reply:
actually this is your code if(content)  when you do the refactor . I think without it, typescript would remind you error

Reply:
My fault, the pr collapsed following code, it is used to add className.